cost/azure/object_storage_optimization/README.md

```diff
@@ -0,0 +1,56 @@
+## Azure Blob Storage Optimization
+ 
+### What it does
+
+This Policy checks Azure Blob Storage for older objects and can move old object to the Cool or Archive tier after a given period of time.
+
+### Functional Details
+
+- This policy identifies all Azure Blob Storage objects last modified outside of the specified timeframe
+- For all objects identified as old, the user can choose to move the object to Cool or Archived tiers after user approval
+ 
+### Input Parameters
+ 
+This policy has the following input parameters required when launching the policy.
+
+- *Azure AD Tenant ID* - the Azure AD Tenant ID used for the Azure API Authentication
+- *Azure Subscription ID* - the Azure Subscription ID used for the Azure API Authentication
+- *Storage Account* - Name of Storage account to search for blobs.
+- *Email addresses to notify* - Email addresses of the recipients you wish to notify when new incidents are created
+- *Move to Cool tier after days last modified* - leave blank to skip moving
+- *Move to Archive tier after days last modified* - leave blank to skip moving
+
+### Prerequesites
+
+- Azure Service Principal (AKA Azure Active Directory Application) with the appropriate permissions to manage resources in the target subscription
+- The following RightScale Credentials
+  - `AZURE_APPLICATION_ID`
+  - `AZURE_APPLICATION_KEY`
+  
+### Required RightScale Roles
+
```

Comment:
add credential_viewer, and observer.

Reply:
Updated readme.md file with the permissions.